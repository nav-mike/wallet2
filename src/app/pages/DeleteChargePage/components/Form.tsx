import styled from 'styled-components/macro';

export const Form = styled.form`
  margin: 1rem 0;
  display: flex;
`;
