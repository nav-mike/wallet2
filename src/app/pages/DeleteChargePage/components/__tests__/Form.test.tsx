import * as React from 'react';
import { render } from '@testing-library/react';
import { Form } from '../Form';

describe('<Form />', () => {
  it('should render abd match the snapshot', () => {
    const form = render(<Form />);
    expect(form.container.firstChild).toMatchSnapshot();
  });
});
