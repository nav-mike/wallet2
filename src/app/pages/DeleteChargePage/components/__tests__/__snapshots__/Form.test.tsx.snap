// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Form /> should render abd match the snapshot 1`] = `
.c0 {
  margin: 1rem 0;
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
}

<form
  class="c0"
/>
`;
